pytests/helpers/session.py

```diff
@@ -128,33 +128,39 @@ def stop(self):
 
         self._wait_for_remaining_output()
 
-    def prepare_to_run(self):
-        """Spawns ptvsd using the configured method, telling it to execute the
-        provided Python file, module, or code, and establishes a message channel
-        to it.
-
-        If use_backchannel is True, calls self.setup_backchannel() before returning.
-
-        If perform_handshake is True, calls self.handshake() before returning.
-        """
-
+    def _get_argv_for_attach_using_import(self):
         argv = [sys.executable]
-        if self.start_method != ('attach', 'pid'):
-            argv += [ptvsd.__main__.__file__]
+        return argv
 
-        if self.start_method == ('attach', 'socket', 'cmdline'):
-            argv += ['--wait']
-        else:
-            self._listen()
-            argv += ['--client']
+    def _get_argv_for_launch(self):
+        argv = [sys.executable]
+        argv += [ptvsd.__main__.__file__]
+        argv += ['--client']
         argv += ['--host', 'localhost', '--port', str(self.ptvsd_port)]
+        if self.multiprocess_port_range:
```

Comment:
We don't have this in the product at the moment (it was there for a while as an artifact of an older multiproc implementation, but it's not needed for the current one).